tests/let/timportc2.nim

```diff
@@ -0,0 +1,8 @@
+discard """
+  errormsg: "'let' symbol requires an initialization"
+  line: "7"
+"""
+
+# Test that this still works when not annotated with importc
+let test: cint
```

Comment:
alternatively, something like
```nim
doAssert not compiles(block:
  let test: cint)
```
to avoid an extra compilation
(at the (IMO) small cost of not checking that exact error msg)